internal/models/sessions.go

```diff
@@ -94,10 +99,13 @@ func FindSessionByID(tx *storage.Connection, id uuid.UUID, forUpdate bool) (*Ses
 		// queries which lock the rows affected by the query from
 		// being accessed by any other transaction that also uses FOR
 		// UPDATE
-		if err := tx.RawQuery(fmt.Sprintf("SELECT id FROM %q WHERE id = ? LIMIT 1 FOR UPDATE;", session.TableName()), id).Exec(); err != nil {
+		if err := tx.RawQuery(fmt.Sprintf("SELECT * FROM %q WHERE id = ? LIMIT 1 FOR UPDATE SKIP LOCKED;", session.TableName()), id).First(session); err != nil {
 			if errors.Cause(err) == sql.ErrNoRows {
+				fmt.Printf("@@@ no session\n")
```

Comment:
can we use the logging library for this? either with the debug / info level

Reply:
ah this was forgotten from me debugging, removing it